src/Diagnostics.DataProviders/Diagnostics.DataProviders.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.2.0" />
```

Comment:
Any reason for using this additional Package?

Reply:
This is to pass the IHostingEnvironment into the ConfigurationFactory constructor.